app/src/main/java/com/sdpteam/connectout/post/view/PostCreatorActivity.java

```diff
@@ -0,0 +1,107 @@
+package com.sdpteam.connectout.post.view;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+import com.sdpteam.connectout.R;
+import com.sdpteam.connectout.post.model.PostFirebaseDataSource;
+import com.sdpteam.connectout.remoteStorage.ImageSelectionFragment;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.widget.Button;
+import android.widget.Switch;
+import android.widget.TextView;
+import androidx.annotation.IdRes;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.appcompat.widget.Toolbar;
+import androidx.fragment.app.FragmentTransaction;
+
+public class PostCreatorActivity extends AppCompatActivity {
+
+    PostCreatorViewModel viewModel;
```

Comment:
Shouldn't it be private ?

Reply:
It should but unfortunately writing tests would be much harder